.buildkite/pipeline_trigger.sh

```diff
@@ -14,11 +14,10 @@ elif [[ "$BUILDKITE_MESSAGE" == *"[pre-release ci]"* ||
   echo "Running pre-release build"
   buildkite-agent pipeline upload .buildkite/pipeline.quick.yml
   buildkite-agent pipeline upload .buildkite/block.full.yml
-elif [[ "$BUILDKITE_MESSAGE" == *"[integration ci]"* ||
+elif [[ "$BUILDKITE_MESSAGE" == *"[quick ci]"* ||
```

Comment:
Is this an intentional change or from when the branch was created initially?

Reply:
Yeah, it's an intentional change to bring it in line with the convention.  I've now also changed the message needed for a "gated full" build.